modules/ROOT/pages/sleep-processor.adoc

```diff
@@ -0,0 +1,32 @@
+= Sleep Processor
```

Comment:
I did an online search, and I can't find any references at all to a sleep processor. I have seen processor sleep mode, but that is about it. Is this something new? Is it a product, feature, component, service? Does it "belong" to Mule?

I guess if I have to ask all these questions, maybe we need a longer introduction?

"A _sleep processor_ is part of <what?>. It is a <blah blah> that enables you to pause...."